### DISCUSSION TITLE
go deps update breaks the build

### DISCUSSION THREAD

**Discussion by lmilbaum** [renovatebot/renovate#23555] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, select which platform you are using.

github.com

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Wanted end result.

go build passes successfully

### What you tried so far.

default.json
```
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "description": "Default preset for use with Renovate's repos",
  "extends": [
    "config:base",
    ":maintainLockFilesWeekly",
    ":enablePreCommit",
    "group:all",
    "regexManagers:dockerfileVersions",
    "schedule:weekly",
    "docker:pinDigests"
  ],
  "packageRules": [
    {
      "matchManagers": [
        "gomod"
      ],
      "matchDepTypes": [
        "indirect"
      ],
      "enabled": true,
      "postUpdateOptions": [
        "gomodTidy"
      ]
    }
  ]
}
```
Renovate generated:
```
-	github.com/pulumi/pulumi-azure-native-sdk/resources v1.104.0
-	github.com/pulumi/pulumi-azure-native-sdk/storage v1.104.0
+	github.com/pulumi/pulumi-azure-native-sdk/resources v2.1.1
+	github.com/pulumi/pulumi-azure-native-sdk/storage v2.1.1
```

should be
```
github.com/pulumi/pulumi-azure-native-sdk/resources/v2 v2.1.1
github.com/pulumi/pulumi-azure-native-sdk/storage/v2 v2.1.1
```

Full renovate log:
https://github.com/platform-engineering-org/renovate-runner/actions/runs/5653783417/job/15315564288

Generated PR:
https://github.com/crc-org/crc-cloud/pull/118

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by lmilbaum** [comment]
bump

**Comment by rarkins** [comment]
There is an artifact error in that PR which you should troubleshoot first

**Reply by lmilbaum** [reply]
Thanks. @rarkins. I'll take a look.

**Reply by lmilbaum** [reply]
I can't figure out why Renovate picked up those artifacts which are failing. They are not listed in the go.mod file.

**Reply by rarkins** [reply]
The `go.sum` checksums include both direct and indirect (transitive) dependencies

**Reply by lmilbaum** [reply]
It would be useful to share the indirect dependencies information in the log

**Reply by rarkins** [reply]
anything which can be shared is shared (stdout, go.sum content)
